lib/src/native/realm_core.dart

```diff
@@ -446,7 +452,7 @@ class LastError {
 }
 
 abstract class Handle<T extends NativeType> {
-  final Pointer<T> _pointer;
+  Pointer<T> _pointer;
```

Comment:
surprised to see final go here :)

Reply:
I think it is a residue from when `releaseEarly` existed. I will add it back,